shopify/resources/inventory_level.py

```diff
@@ -5,8 +5,13 @@
 
 class InventoryLevel(ShopifyResource):
 
-    def __repr__(self):
-        return '%s(%s)' % (self._singular, self.available)
+    @classmethod
+    def _element_path(cls, prefix_options={}, query_options=None):
+        if query_options is None:
+            prefix_options, query_options = cls._split_options(prefix_options)
+
+        return "%s%s.%s%s" % (cls._prefix(prefix_options)+'/', cls.plural,
+                              cls.format.extension, cls._query_string(query_options))
```

Comment:
Is there a way to have more informative way for the object representation than this `inventory_level(None)`, the inventory does not have it's own id? Clearly overriding the `__repr__` seems wrong approach.

Reply:
FYI, this is where I landed

>>> x = shopify.InventoryLevel.find(inventory_item_ids=16428756291, location_ids=13968638008)
>>> x
[inventory_level(inventory_item_id=16428756291, location_id=13968638008)]